pkg/httpclient/httpclient.go

```diff
@@ -0,0 +1,30 @@
+package httpclient
+
+import (
+	"fmt"
+	"net/http"
+)
+
+var Version string
```

Comment:
This version string should default to a message that says that the version wasn't compiled into the binary.

I'd also prefer that this be a completely separate package so that any part of the code can import the clair version.

See: https://github.com/rkt/rkt/blob/master/version/version.go